requirements/common.txt

```diff
@@ -1,8 +1,5 @@
 alembic==0.7.6
-async==0.6.2
```

Comment:
in requirements file you add the Python packages that this app depends on (i.e. dependencies). In other words, all those that you install with `pip install X`.